internal/tiltfile/tiltfile_test.go

```diff
@@ -1804,7 +1804,9 @@ func TestYamlErrorFromHelm(t *testing.T) {
 	f.file("Tiltfile", `
 k8s_yaml(helm('helm'))
 `)
-	f.loadErrString("from helm")
+
+	// TODO(dmiller): there should be a better assertion here
+	f.loadErrString("helm")
```

Comment:
Yeah this test is silly and asserts on the output of `helm`. Any ideas?

Reply:
i don't think that's silly. an easy mistake to make is to accidentally swallow the output from the tool

have you considered using the getHelmVersion function, and have different asserts per version?

Reply:
That's a good idea, I'll do that!